### PR TITLE
test: update snapshot for `webpack-dev-server@4.0.0`

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
test, ci fix

**Did you add tests for your changes?**
updated snapshot
**If relevant, did you update the documentation?**
No
**Summary**

CI is failing for `webpack-dev-server@next`

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

**Review by alexander-akait — APPROVED** [review]

**Comment by alexander-akait** [comment]
In future we will allow to run multiple server dev server

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -56,10 +56,10 @@ jobs:
         os: [ubuntu-latest, windows-latest, macos-latest]
         node-version: [10.x, 12.x, 14.x, 16.x]
         webpack-version: [4, latest]
-        dev-server-version: [latest, next]
+        dev-server-version: [version-3, latest]
```
Comment (anshumanv) [inline .github/workflows/nodejs.yml:59]: let's just remove v3 snapshots? They are already tested

Comment (snitin315) [reply .github/workflows/nodejs.yml:59]: We will drop `webpack-dev-server@3` support for `webpack-cli@next`

Comment (anshumanv) [reply .github/workflows/nodejs.yml:59]: let's drop them in next then

**Comment by snitin315** [comment]
`macOS` and `ubuntu` have different snapshots 😞

**Comment by alexander-akait** [comment]
Problem here in `version-3`, it was point on `3.11.1` version, not `3.11.2`, fixed in npm

**Pull request merged**